test/examples.ts

```diff
@@ -1,9 +1,12 @@
 /* tslint:disable:no-console */
 
 import { assert } from 'chai';
-import * as fs from 'fs';
-import * as path from 'path';
-import { EndOfStreamError, StreamReader } from '../lib';
+import fs from 'node:fs';
+import path from 'node:path';
+import { EndOfStreamError, StreamReader } from '../lib/index.js';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
Tip for line 15: `fs.createReadStream` can also accept a `URL`, so it could be `fs.createReadStream(new URL('resources/JPEG_example_JPG_RIP_001.jpg', import.meta.url))`.

Reply:
Done as suggested